client/influxdb.go

```diff
@@ -338,6 +359,10 @@ func (p *Point) MarshalJSON() ([]byte, error) {
 	return json.Marshal(&point)
 }
 
+func (p *Point) MarshalString() string {
+	return tsdb.NewPoint(p.Name, p.Tags, p.Fields, p.Time).String()
```

Comment:
Yeah, was trying avoid re-implementing the serialization again.  Might be able to re-work this to avoid creating the extra point though.